libs/agno/agno/memory/manager.py

```diff
@@ -291,6 +296,69 @@ def delete_user_memory(
             log_warning("Memory DB not provided.")
             return None
 
+    def clear_user_memories(self, user_id: Optional[str] = None) -> None:
+        """Clear all memories for a specific user.
+
+        Args:
+            user_id (Optional[str]): The user id to clear memories for. If not provided, clears memories for the "default" user.
+        """
+        if user_id is None:
+            log_warning("Using default user id.")
+            user_id = "default"
+
+        if not self.db:
+            log_warning("Memory DB not provided.")
+            return
+
+        if isinstance(self.db, AsyncBaseDb):
+            raise ValueError(
+                "clear_user_memories() is not supported with an async DB. Please use aclear_user_memories() instead."
+            )
+
+        # Get all memories for the user
+        memories = self.get_user_memories(user_id=user_id)
```

Comment:
This is very inefficient. We should extend `delete_user_memories` to not require a list of memory_ids, but just the user ID. 
Please add a TODO here, because that is out of scope for this task